composer.json

```diff
@@ -14,7 +14,7 @@
   ],
   "require": {
     "php": "^5.6|^7.0",
-    "cache/array-adapter": "^0.4.0",
+    "cache/array-adapter": "^0.4.2",
```

Comment:
Should be updated! As you can see here: https://github.com/php-cache/adapter-common/commit/a1361f7f8b8557f6e6392131f8078a189e345b40#diff-fbe933f438ac3d91fb49186a962c4978

The Caching has been switched from DateTime to Timestamp. This should be updated to prevent errors